CellDataCoupler/Classes/Extensions/BundleExtensions.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  BundleExtensions.swift
+//  CellDataCoupler
+//
+//  Created by Jangle's MacBook Pro on 10/29/20.
+//
+
+import Foundation
+
+internal extension Bundle {
```

Comment:
This could go too

Reply:
After talking, we'll close this in favor of using a zero-height empty cell if a coupler isn't successfully loaded (the cell has to come from the cocoapod's bundle)